### ISSUE TITLE
"Fuel / 10000 mi" string is wrong.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [evancharlton/android-mileage#3]
```
What steps will reproduce the problem?
1. Open Statistics tab

What is the expected output? What do you see instead?
"Fuel / 10000 mi" shows as "Fuel / 10000 10000"

Please use labels and text to provide additional information.

```

Original issue reported on code.google.com by `evancharlton` on 20 Nov 2010 at 12:13

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Mileage 3.0
```

Original comment by `evancharlton` on 30 Nov 2010 at 12:55
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
Original comment by `evancharlton` on 17 Apr 2011 at 8:21
- Changed state: **Released**
